arangod/RocksDBEngine/RocksDBRecoveryManager.cpp

```diff
@@ -133,6 +134,15 @@ class WBReader final : public rocksdb::WriteBatch::Handler {
 
   // sequence number from which we start recovering
   rocksdb::SequenceNumber const _recoveryStartSequence;
+  // latest sequence in WAL
+  rocksdb::SequenceNumber const _latestSequence;
+
+  // informational section, used only for progress reporting
+  rocksdb::SequenceNumber _sequenceRange = 0;
+  rocksdb::SequenceNumber _rangeBegin = 0;
+  int _reportTicker = 0;
+  int _progress = 0;
```

Comment:
Because otherwise it's not clear that this is only related to logging

```suggestion
struct ProgressState {
  // latest sequence in WAL
  rocksdb::SequenceNumber const _latestSequence;

  // informational section, used only for progress reporting
  rocksdb::SequenceNumber _sequenceRange = 0;
  rocksdb::SequenceNumber _rangeBegin = 0;
  int _reportTicker = 0;
  int _progress = 0;
};
```

Reply:
Addressed